byteosaurus_hex.py

```diff
@@ -1495,6 +1500,10 @@ def icmp_packet(fuzzy, module, icmp_type, icmp_inputs):
                 src_mac, dst_mac = RandMAC()._fix(), RandMAC()._fix()
                 src_ip, dst_ip = RandIP("172.16.0.0/12")._fix(), RandIP("172.16.0.0/12")._fix()
                 ttl = randint(10, 255)
+            if module == 'MPLS':
```

Comment:
considering VXLAN and MPLS share the same sections, we can keep it as an `or`

```python
if module == 'VXLAN' or module == 'MPLS'
```

Reply:
Addressed